tests/components/cert_expiry/test_config_flow.py

```diff
@@ -48,7 +48,7 @@ async def test_user_with_bad_cert(hass: HomeAssistant) -> None:
     assert result["step_id"] == "user"
 
     with patch(
-        "homeassistant.components.cert_expiry.helper.get_cert",
+        "homeassistant.components.cert_expiry.helper.async_get_cert",
```

Comment:
Its not great to patch this, but its not so great to patch `create_connection` instead so I left the pattern as-is